crates/matrix-sdk/src/event_cache/mod.rs

```diff
@@ -646,14 +646,14 @@ impl RoomEventCacheInner {
     async fn append_events_locked_impl(
         &self,
         mut room_events: RwLockWriteGuard<'_, RoomEvents>,
-        events: Vec<SyncTimelineEvent>,
+        timeline_events: Vec<SyncTimelineEvent>,
```

Comment:
I don't like this renaming either, there's something called `TimelineEvent` and the name suggests it's a collection of such `TimelineEvent`. Please keep it as `events` or use the full type in the name, aka `sync_timeline_events`.

Reply:
`sync_timeline_events` is better then.